src/main/kotlin/org/wycliffeassociates/otter/jvm/app/ui/projecthome/view/ProjectHomeView.kt

```diff
@@ -44,46 +46,48 @@ class ProjectHomeView : View() {
                 leftAnchor = 0
                 rightAnchor = 0
             }
-            content = flowpane {
-                addClass(AppStyles.appBackground)
-                addClass(ProjectHomeStyles.projectsFlowPane)
-                bindChildren(viewModel.projects) {
+            content =
                     hbox {
-                        projectcard(it) {
-                            addClass(ProjectHomeStyles.projectCard)
-                            titleLabel.addClass(ProjectHomeStyles.projectCardTitle)
-                            languageLabel.addClass(ProjectHomeStyles.projectCardLanguage)
-                            cardButton.apply {
-                                text = messages["loadProject"]
-                                action {
-                                    viewModel.openProject(it)
-                                }
-                            }
-                            deleteButton.apply {
-                                action {
-                                    error(
-                                            messages["deleteProjectPrompt"],
-                                            messages["deleteProjectDetails"],
-                                            ButtonType.YES,
-                                            ButtonType.NO,
-                                            title = messages["deleteProjectPrompt"]
-                                    ) { button: ButtonType ->
-                                        if (button == ButtonType.YES) {
-                                            viewModel.deleteProject(it)
-                                            cardButton.isDisable = true
-                                            isDisable = true
+                        addClass(AppStyles.appBackground)
+                        flowpane {
+                            hgrow = Priority.ALWAYS
+                            addClass(AppStyles.appBackground)
+                            addClass(ProjectHomeStyles.projectsFlowPane)
+                            bindChildren(viewModel.projects) {
+                                hbox {
+
+                                    add(card {
+                                        addClass(DefaultStyles.defaultCard)
+                                        cardfront {
+                                            isComplete = true
+                                            var cardGraphic = ImageLoader.load(
+                                                    ClassLoader.getSystemResourceAsStream("images/project_image.png"),
+                                                    ImageLoader.Format.PNG
+                                            )
```

Comment:
the ImageLoader feels like it doesn't belong in the view

Reply:
i can move it out into the AppStyles similar to how we are grabbing icons?